pages/common/jigsaw.md

```diff
@@ -0,0 +1,28 @@
+# jigsaw
+
+> A Laravel-based static site builder for PHP.
+> More information: <https://jigsaw.tighten.co>.
+
+- Initialise a project:
+
+`jigsaw init`
+
+- Initialise a project using a starter template:
```

Comment:
```suggestion
- Initialise a project using a starting template:
```

Reply:
Maybe this makes more sense?

Reply:
Actually "starter", being an adjective, is more appropriate here.

Reply:
Makes sense 👍